packages/switch/README.md

```diff
@@ -58,6 +58,46 @@ focus to the content.
 </div>
 ```
 
+## Sizes
+
+<sp-tabs selected="m" auto label="Size Attribute Options">
+
+<sp-tab value="s">Small</sp-tab>
+<sp-tab-panel value="s">
+
+```html
+<sp-switch size="s">Small</sp-switch>
+```
+
+</sp-tab-panel>
+
+<sp-tab value="m">Medium</sp-tab>
+<sp-tab-panel value="m">
+
+```html
+<sp-switch size="m">Medium</sp-switch>
+```
+
+</sp-tab-panel>
+
+<sp-tab value="l">Large</sp-tab>
+<sp-tab-panel value="l">
+
+```html
+<sp-switch size="l">Large</sp-switch>
+```
+
+</sp-tab-panel>
+
+<sp-tab value="xl">Extra Large</sp-tab>
+<sp-tab-panel value="xl">
+
+```html
+<sp-switch size="xl">Extra Large</sp-switch>
+```
+
+</sp-tab-panel>
+</sp-tabs>
```

Comment:
I buggered this. There has to be a blank line between this and the next text content.

Reply:
There _may_ be more, but you can also check locally via `yarn docs:start`...

Reply:
yeah, ran the docs:start, getting:
´´´
rollup v2.77.2
bundles  → dist...
created dist in 25s

[2022-09-13 15:09:45] waiting for changes...
´´´

Markup Preview looks good IMO. 
<img width="1079" alt="Screen Shot 2022-09-13 at 15 22 39" src="https://user-images.githubusercontent.com/52184321/190011716-5c47de0b-ef85-403e-934c-7af7cd2001db.png">

Reply:
Oh, it usually opens the docs in the browser, but sometimes browser versions or plugins prevent it. There's some issue in the markdown parser with HTML that doesn't like to white spaces above. I'll point them out.

Reply:
> Oh, it usually opens the docs in the browser, but sometimes browser versions or plugins prevent it. There's some issue in the markdown parser with HTML that doesn't like to white spaces above. I'll point them out.

Gotcha! I have made the suggested changes.

Reply:
Awesome. Had no idea the markdown parser was so fragile, but it look like it's working now: https://bschmidt-switch-core-tokens--spectrum-web-components.netlify.app/components/switch/#sizes Thanks!!